src/test/java/com/google/googleidentity/oauth2/endpoint/AuthorizationEndpointTest.java

```diff
@@ -94,14 +87,9 @@ public void init() {
     userDetailsService.addUser(USER);
     userSession = new UserSession();
     userSession.setUser(USER);
-    Map<GrantType, RequestHandler> map = new HashMap<>();
```

Comment:
Can the test cases pass without setup the expected respond from the mock?

Reply:
They can, actually I did not test the handlers here.